platform.md

```diff
@@ -212,6 +224,19 @@ A platform MAY support any number of mixins for a given stack in order to suppor
 Changes introduced by mixins SHOULD be restricted to the addition of operating system software packages that are regularly patched with strictly backwards-compatible security fixes.
 However, mixins MAY consist of any changes that follow the [Compatibility Guarantees](#compatibility-guarantees).
 
+#### Mixin Resolution
+
+During the prepare phase, a list of provided mixins MUST be retrived from the provided run-image. The preparer should either fail or ignore the previous image if the run-image is missing any mixins that were present on the previous image.
```

Comment:
When would you ignore vs just fail if it's missing any mixins?

Reply:
I think it would be up to the platform. But /cc @ekcasey and @sclevine

Reply:
@jkutner do you have an example when a platform would fail if there's a mismatch for a platform?